dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/compiler/JDTCompiler.java

```diff
@@ -6,9 +6,6 @@
  * http://www.eclipse.org/legal/epl-2.0/
```

Comment:
What date should be used for the date range in this class? Was this one changed by Evan or is this new 2023 code?

Reply:
Evans's changes from 2021. I don't think I touched it.

Reply:
If the file wasn't touched then keeping the original date from Evan is the right thing to do if it was pushed up back in 2021.